soh/src/overlays/actors/ovl_En_Elf/z_en_elf.c

```diff
@@ -1421,11 +1421,11 @@ void func_80A053F0(Actor* thisx, PlayState* play) {
     } else {
         this->actionFunc(this, play);
         thisx->shape.rot.y = this->unk_2BC;
-        nREG(80) = gSaveContext.sceneFlags[127].chest;
+        nREG(80) = HIGH_SCORE(HS_HBA);
```

Comment:
This is an OOB that we now have to fix. This is an odd way to access a highscore, and decomp went with this because of the debug comment. Decomp has a comment explaining it now, but for SceneDB, we have to fix this.